rest-assured-tests/src/test/java/TariffManagementServiceClass.java

```diff
@@ -0,0 +1,39 @@
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.*;
+import static org.junit.Assert.assertEquals;
+
+public class TariffManagementServiceClass {
+
+    static Tariff[] getTariffsList() {
+        Response response = get("/tariffs");
+        assertEquals(200, response.getStatusCode());
+        return Tariff.jsonToTariffsList(response);
+    }
+
+    static Tariff getTariffById(int id) {
+        Response response = get("/tariffs/" + id);
+        assertEquals(200, response.getStatusCode());
+        return Tariff.jsonToTariff(response);
+    }
+
+    static Response createTariffByPost(String name, float price, int washingTime, int statusCode) {
+        Response response = given()
+                .body("{\n" +
```

Comment:
We can do this by passing/creating TariffDto and converting it to json

Reply:
I'm not sure that is a good practice to use code of application in test suite, cause it should be isolated.